src/x86_64/pvclock.c

```diff
@@ -0,0 +1,39 @@
+#include <runtime.h>
```

Comment:
Any reason why this is sitting in its own file rather than kvm_platform.c?

Reply:
Yes, they both use pvclock, so I split it out.